tests/unit/other.tcl

```diff
@@ -375,13 +375,13 @@ start_server {tags {"other external:skip"}} {
         waitForBgsave r
 
         # Hash table should rehash since there is no child process,
-        # size is power of two and over 4098, so it is 8192
+        # size is power of two and over 4096, so it is 8192
         wait_for_condition 50 100 {
             [string match "*table size: 8192*" [r debug HTSTATS 9]]
         } else {
             fail "hash table did not rehash after child process killed"
         }
-    } {} {needs:debug needs:local-process}
+    } {} {needs:debug}
```

Comment:
that's because we do `kill -9`
```suggestion
    } {} {needs:debug needs:local-process}
```

Reply:
if they have `external:skip` then i suppose it gets the same purpose.
not sure we have a reason to invest effort in keeping this part clean if it doesn't have an impact (breaks anything).
i.e. it'll easily get un-clean again over time.